.github/workflows/build.yml

```diff
@@ -163,10 +163,13 @@ jobs:
           export CUDNN_DIR=/usr/local/cuda
           export CUDA_DIR=/usr/local/cuda
           export FF_HOME=$(pwd)
+          cd $FF_HOME/deps/tokenizers-cpp/example
```

Comment:
is there a way to avoid having to do this manually, e.g. by running these instructions from the Makefile as shell commands (see for example https://stackoverflow.com/questions/10024279/how-to-use-shell-commands-in-makefile)?